src/components/ConsentPicker.vue

```diff
@@ -0,0 +1,486 @@
+<template>
+  <PvPanel class="m-0 p-0 w-full" header="Select params for the Consent and Assent forms">
+    <div class="card flex justify-content-center">
+      <div class="flex flex-wrap gap-3">
+        <div class="flex align-items-center">
+          <PvRadioButton v-model="decision" inputId="helpChoose" name="help" value="help" @change="whatDecision" />
+          <label for="helpChoose" class="ml-2">Help me choose</label>
+        </div>
+        <div class="flex align-items-center">
+          <PvRadioButton v-model="decision" inputId="iKnow" name="know" value="know" @change="whatDecision" />
+          <label for="iKnow" class="ml-2">I know what to select</label>
```

Comment:
Could we change this to `Let me choose`? I feel that it is a better complement to `Help me choose`.

Reply:
I think for now it should be different enough for people to select the correct one :) But could be changed